node/src/BaseClient.ts

```diff
@@ -516,6 +512,77 @@ function getRequestErrorClass(
     return RequestError;
 }
 
```

Comment:
Moved the function here from node/src/GlideClusterClient.ts so that it can be used in createWritePromise()

Reply:
```
@internal
```
or export?

Reply:
Updated. Added @internal 
export is not required, as it is only used in this file. We can add later if this function has to be used outside.